web/datasetAPI/hidden_endpoints_test.go

```diff
@@ -181,6 +181,33 @@ func TestPublishingEndpointsAreHiddenForWeb(t *testing.T) {
 			})
 		})
 
+		// POST request to /instances/{instance_id}/dimensions
+		Convey("When a POST request to create a dimension option resource for an instance in web", func() {
+			Convey("Then response returns a status not found (404)", func() {
+
+				log.Debug("POST request on Dimension Option resource for an instance", log.Data{"endpoint": "/instances/{instance_id}/dimensions", "method": "POST"})
+
+				datasetAPI.POST("/instances/{instance_id}/dimensions", instanceID).
+					WithHeader(florenceTokenName, florenceToken).
+					WithBytes([]byte(validPOSTAgeDimensionJSON)).
+					Expect().Status(http.StatusNotFound).
+					Body().Contains("")
```

Comment:
shouldn't this be `Empty()` or similar? or not there (is it actually checking anything?)